### DISCUSSION TITLE
How to set app always fix on the top(nearby the tray icon) at launch

### DISCUSSION THREAD

**Discussion by Jun10ng** [tauri-apps/tauri#6023] [Q&A] [answered]
Hi master, 
I am new here, I want to set my app fix on the top(nearby the tray icon), **including the first-time launch of my app.**
Which API I should use?

In my current implementation, I call `hide()` at app launch.

**Comment by JonasKruckenberg** [comment, marked as answer]
There is the [`tauri-plugin-positioner`](https://github.com/tauri-apps/plugins-workspace/tree/dev/plugins/positioner) that can help with tray relative positions (only works after the user clicked on the tray icon once though) so what I would do is the following (if that works for your use case):
1. Have the app be `"visible": false` at the start
2. Once the user clicks the tray icon once we know its position and can position the window
3. Have a system-tray button that says "Open UI" or something like this
4. In the buttons callback you call `let _ = win.move_window(Position::TrayCenter);` and then `win.show()`

**Reply by Jun10ng** [reply]
thanks, it's work! 
PS: for the people who have same question. 
```
[dependencies]
tauri-plugin-positioner = {version = "1.0", features = ["system-tray"] }
```
and set [visible prop](https://tauri.app/v1/api/config/#windowconfig)

**Reply by kurdin** [reply]
Is they a way to add position offsets for `Position::TrayCenter`? It opens too close to system tray, I want to move it a bit up (windows) or down (mac)

**Reply by DaniBedz** [reply]
Wondering this also.
